pandas/_libs/parsing.pyx

```diff
@@ -0,0 +1,682 @@
+# -*- coding: utf-8 -*-
+# cython: profile=False
```

Comment:
this should be in tslibs/parsing

Reply:
how is it possible that ci is green given that all references in setup.py refer to tslibs/parsing.pyx ?

Reply:
Ah, the file is just duplicated in the diff, that's why it still works :-). 
@jbrockmendel so this file can just be removed (if they are identical)

Reply:
To get asv to run a while back I had to move tslibs/parsing.pyx to just parsing.pyx.  This is an artifact that shouldn't have gotten pushed.  Will remove.